tools/gh-rename-branches.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+BASE=$(cd $(dirname "${BASH_SOURCE[0]}")/.. && pwd)
+
+echo "This script is set to rename the default branch only on test repositories by default. Please open the file and \
+insert the correct variable name - REPOSITORIES - in the for loop to run the rename script."
+echo -e "If the default branch is not master and no master branch exists, \
+no renaming will happen in that particular repository. \n"
+
+echo -e "\033[1mWhat is your GitHub personal access token?\033[0m See here for more: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token >"
+
+read usertoken
+
+TESTREPOSITORIES=(jetpack-google-analytics-helper jptest-tools jetpack-gh-testing)
+
+# REPOSITORIES=(jetpack-changelogger remove-asset-webpack-plugin jetpack-compat babel-plugin-replace-textdomain \
+# i18n-loader-webpack-plugin i18n-check-webpack-plugin jetpack-config action-push-to-mirrors action-pr-is-up-to-date \
+# eslint-changed eslint-config-target-es jetpack-analyzer jetpack-always-use-jetpack-open-graph jetpack-error \
+# jetpack-shared-extension-utils jetpack-debug-helper jetpack-base-styles jetpack-waf jetpack-a8c-mc-stats \
+# jetpack-autoloader jetpack-logo jetpack-device-detection jetpack-roles ignorefile jetpack-google-fonts-provider \
+# jetpack-constants jetpack-composer-plugin jetpack-admin-ui jetpack-password-checker jetpack-blocks jetpack-codesniffer \
+# action-required-review action-repo-gardening vaultpress jetpack-beta jetpack-plugins-installer jetpack-status \
+# jetpack-redirect jetpack-assets jetpack-post-list jetpack-connection jetpack-lazy-images jetpack-tracking \
+# jetpack-heartbeat jetpack-options jetpack-licensing jetpack-abtest jetpack-partner jetpack-publicize \
+# jetpack-identity-crisis jetpack-wordads jetpack-jitm jetpack-sync jetpack-my-jetpack jetpack-connection-ui \
+# jetpack-search jetpack-starter-plugin jetpack-protect-plugin jetpack-search-plugin jetpack-backup \
+# jetpack-boost-production jetpack-backup-plugin jetpack-storybook jetpack-production)
+
+
+# Renaming the default branch in a list of given repositories, using CURL
+
+for REPOSITORY in "${TESTREPOSITORIES[@]}"
+    do
```

Comment:
Your indenting is weird here, and uses spaces instead of tabs that we use everywhere else.

Personally I'd have it look something like
```bash
for REPOSITORY in "${TESTREPOSITORIES[@]}"; do
	echo "Renaming master to trunk on $REPOSITORY"

	curl \
		-X POST \
		etc...
done
```
i.e. treating the "do" and "done" like `{` and `}` in PHP or JS, and indenting the broken line an extra level.